.claude/skills/ship-release/SKILL.md

```
@@ -0,0 +1,110 @@
+2. `git fetch origin master` and confirm the merge commit is HEAD of `origin/master` and its `package.json` has TARGET.
+3. **GATE: confirm with the user before pushing the tag** (tag push = build + release creation; deleting a tag after builds start is messy).
+4. Move the release worktree HEAD onto the master merge commit, then tag via the repo script. The release worktree is still on `chore/release-<version>` (the pre-merge bump commit) — tagging there ships the wrong tree. Detach onto the squashed merge commit first (its `package.json` version must equal TARGET):
+   ```
+   MERGE_SHA=$(git rev-parse origin/master)
+   git checkout "$MERGE_SHA"          # detached HEAD at the merge commit
+   node -p "require('./package.json').version"   # MUST print TARGET
+   yarn release:tag                    # reads package.json version, guards, tags HEAD, pushes
+   ```
+   `yarn release:tag` (`scripts/release-tag.ts`) reads the version from `package.json`, refuses if the tag already exists or isn't greater than the latest tag in-channel, then tags the current HEAD as the bare version and pushes it. It prompts `Proceed? (y/N)` — pipe `y` for non-interactive (`echo y | yarn release:tag`).
+   - **node_modules required**: a fresh worktree has none, so `yarn release:tag` fails with `Couldn't find the node_modules state file (findPackageLocation)`. Either run `yarn install` in the worktree first, or run the script's exact equivalent by hand after verifying its guards yourself:
+     ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)